src/sentry/south_migrations/0284_auto__del_field_release_project__add_field_release_project_id__chg_fie.py

```diff
@@ -724,7 +699,7 @@ def backwards(self, orm):
             'id': ('sentry.db.models.fields.bounded.BoundedBigAutoField', [], {'primary_key': 'True'}),
             'is_verified': ('django.db.models.fields.BooleanField', [], {'default': 'False'}),
             'user': ('sentry.db.models.fields.foreignkey.FlexibleForeignKey', [], {'related_name': "'emails'", 'to': "orm['sentry.User']"}),
-            'validation_hash': ('django.db.models.fields.CharField', [], {'default': "u'R6zOiWKMO0GUE2e6HGMyFwPxDqc7hR7E'", 'max_length': '32'})
+            'validation_hash': ('django.db.models.fields.CharField', [], {'default': "u'8g5HTkl1az1PBdiGQi2Ekji7kGyTFL9J'", 'max_length': '32'})
```

Comment:
lol this is actually bad, this is a bug in our side that has somehow never caused a problem